locale/pt-br/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Recomendado Para Maioria dos Usuários
   changelog: Changelog
   api: Documentação da API
-  version-schedule-prompt: Ou verifique o
-  version-schedule-prompt-link-text: Cronograma de versões LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```

Comment:
@nodejs/nodejs-pt  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?